opae-libs/cmake/modules/OPAE.cmake

```diff
@@ -52,6 +52,7 @@ if(NOT libuuid_FOUND)
 endif()
```

Comment:
This file lives in opae-libs, so it must be updated at https://github.com/OPAE/opae-libs.git.

Reply:
Done the changes and pushed to opae-libs.git